mlir/test/Dialect/Vector/vector-multi-reduction-inner-parallel-pass-lowering.mlir

```diff
@@ -0,0 +1,39 @@
+// RUN: mlir-opt -lower-vector-multi-reduction="lowering-strategy=inner-parallel" -split-input-file %s | FileCheck %s
+
+// -----
```

Comment:
This is the separator for tests, so we don't need it before any tests.

Reply:
Thanks, the test file is updated.